src/end-to-end/edge-end-to-end.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+# Recommendation: Do not clone the ts-client-e2e-tests repo in local ts-client repo; it'll just cause npm deps issues
+
+# inputs
+clone=$1 # required -- `y` or `n` to clone the ts-client-e2e-tests repo
```

Comment:
This feels a little confusing considering the script has an option to clone or not clone. I asked a question about this previously but I feel like I'm missing something.